<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ActorSetupPage</class>
 <widget class="QWizardPage" name="ActorSetupPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <property name="title">
   <string>Descrição da gravação</string>
  </property>
  <property name="subTitle">
   <string>Preencha os campos abaixo com detalhes referentes ao que sera gravado.</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <property name="margin">
      <number>10</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Ator:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QWizardCombobox" name="cmbActor">
       <property name="duplicatesEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Palavra:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QWizardCombobox" name="cmbWord"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Tomada:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="spinTake">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>5</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWizardCombobox</class>
   <extends>QComboBox</extends>
   <header location="global">qwizardcombobox.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
